### ISSUE TITLE
Possibility to train other models/networks rather than DetectNet with DIGITS

### ISSUE THREAD

**Issue by VanitarNordic** [NVIDIA/DIGITS#1398]
Hello,

if the DIGITS is a GUI for CAFFE, then is it possible to train SSD (https://github.com/weiliu89/caffe/tree/ssd) or other CAFFE based networks on it?

**Comment by lukeyeager** [comment]
> DIGITS requires a build of Caffe. We officially only support recent releases from NVIDIA/caffe (NVcaffe), but any recent build of BVLC/caffe will probably work too.
https://github.com/NVIDIA/DIGITS/blob/digits-5.0/docs/BuildCaffe.md#building-caffe

You're on your own trying to use other forks of Caffe.

**Comment by VanitarNordic** [comment]
Yes, I meant any model which handles a `caffemodel` and a `prototxt` files. Then the DIGITS should support it by default, yes?

Besides, Yes I have used DIGITS for a BVLC-Caffe based model and it was working properly.

But for creating a dataset for a model (any object detection model), I think we have only use KITTI format, yes?

**Comment by VanitarNordic** [comment]
Also, have you heard about YOLO9000?

**Comment by jaydee713** [comment]
@VanitarNordic  Did you end up getting SSD to work with DIGITS?

**Comment by VanitarNordic** [comment]
@jaydee713

No, because SSD has modified the original Caffe for its own application.

**Comment by MohammadMoradi** [comment]
Hi,
I think it would be a good idea to add support for SSD since it is the state of art network for object detection. I want to train SSD on my own data by DIGITS but it seems that "AnnotatedData" layer is not supported. I think the only thing needed to be done is to add AnnotatedData to DIGITS or at least to implement it in python.

**Comment by PiyalGeorge** [comment]
@dqthebt24  @VanitarNordic , Can you tell me how you did that training? I have a mobileNet-SSD caffemodel and deploy.prototxt, which i trained custom with my own dataset and is working fine. But i dont know how to use it with DIGITS or train with DIGITS. 
Please help me.
